client/executor/wasmtime/Cargo.toml

```diff
@@ -10,16 +10,15 @@ description = "Defines a `WasmRuntime` that uses the Wasmtime JIT to execute."
 
 [dependencies]
 log = "0.4.8"
-wasmi = "0.6.2"
+scoped-tls = "1.0"
 parity-wasm = "0.41.0"
 codec = { package = "parity-scale-codec", version = "1.0.0" }
 sc-executor-common = { version = "0.8.0-alpha.2", path = "../common" }
 sp-wasm-interface = { version = "2.0.0-alpha.2", path = "../../../primitives/wasm-interface" }
 sp-runtime-interface = { version = "2.0.0-alpha.2", path = "../../../primitives/runtime-interface" }
 sp-core = { version = "2.0.0-alpha.2", path = "../../../primitives/core" }
 sp-allocator = { version = "2.0.0-alpha.2", path = "../../../primitives/allocator" }
-
-wasmtime = "0.11"
+wasmtime = { git = "https://github.com/paritytech/wasmtime", branch = "a-thread-safe-api" }
```

Comment:
this will prevents publishing on crates.io (though not sure if it is a bad thing) for a while

Reply:
yes, it will prevent us from publishing this on crates.io . Current plan is to get this rolled out on kusama, confirm the findings, get upstream merged and switch back to the newly released version.